data/projects/vulture.md

```diff
@@ -1,57 +1,126 @@
-Vulture is a tool that finds unused classes, functions and variables in your
-code.
-This information is useful as unused code can be removed without changing the
-behavior of a program while making it easier to maintain the remaining code
-and slimming down a codebase.
-Using the information from vulture, one would normally remove the dead code
-manually.
-But similar to auto formatting tools it would save a lot of work if there was
-a way to remove the code automatically.
-
-This project's aim is to provide such an auto removal feature via a coala bear,
-using the information vulture offers.
-
-To reach this goal the VultureBear should be modified to support removing a
-source range that vulture declared dead code.
-
-To realize this, some work has to be done upstream at vulture. Namely:
-
-- Make vulture available as a library for easier use for the bear
-- Implement a confidence value for results. This is necessary as vulture
-  produces false positives so a threshold has to be set what to accept for auto
-  removal.
-- Make vulture offer source ranges for the dead code. Right now vulture only
-  offers the beginning of the dead code, which makes removal hard. This could
-  possibly be done by removing everything up to the next node in the python AST.
-  Another possibility is that the enhanced AST of pyflakes could offer the
-  necessary information.
-- In case no way to offer source ranges is found, additional detection cases
-  can be implemented for vulture. For example identifying `if False` cases.
+#### Overview
+The motivation, here, is to extend the functionality of vulture as a library, and to pass on all metadata through the API
```

Comment:
Only one violation, nested-lists are being treated as an indented code block:
```diff
|  29|    |-	- Refactor VultureBear to directly fetch results through get_unused_code (API), thus making it more efficient- we would have memory files passing (An extra layer of parsing would then be removed)
|  30|    |-	- Further enhancements in vulture (detect unreachable code and reporting ranges of dead code)  would influence the API, which would also need refactoring of the Bear.
|  31|    |-
|    |  29|+  ```
|    |  30|+  	- Refactor VultureBear to directly fetch results through get_unused_code (API), thus making it more efficient- we would have memory files passing (An extra layer of parsing would then be removed)
|    |  31|+  	- Further enhancements in vulture (detect unreachable code and reporting ranges of dead code)  would influence the API, which would also need refactoring of the Bear.
|    |  32|+  ```

```
I hope this can be ignored for now.